### PR TITLE
Rename/aicspylibczi

### PR DESCRIPTION
**Pull request by heeler**
After renaming the project for release I decided to change the repo name to mirror the package name. 
 - The docs now build automatically on the particular branches ( tag trigger builds don't seem to work )
 - I changed the python read_image to take an M dimension
 - I refreshed all docs 

The only code changes are in CziFile.py all other changes are in the documentation and build setup

### CODE REVIEW

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -3,73 +3,40 @@ name: Docs
 on:
   push:
     branches:
-      - stable
+      - rename/aicspylibczi
+    tags:
+      - 'docs'
```
Comment (JacksonMaxfield) [inline .github/workflows/build-docs.yml]: Change this to master without the tags before merging.

File: .github/workflows/build-docs.yml [hunk 2]
```diff
@@ -3,73 +3,40 @@ name: Docs
...
           cmake ..
           cmake --build . --target docs
-          cd $FOLDER
-          touch .nojekyll
-          git add -f .nojekyll
-          git add ./*
-          git commit -m "Deploying branch to gh-pages" --quiet
-          git push $REPO_PATH ${BRANCH} --quiet
+          cd docs/html
+          pwd
```
Comment (JacksonMaxfield) [inline .github/workflows/build-docs.yml]: Don't think you need these last two

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# pylibczi
-[![C++ Build & Test](https://github.com/AllenCellModeling/pylibczi/workflows/C%2B%2B%20Master%20Build/badge.svg)](https://github.com/AllenCellModeling/pylibczi/actions)
-[![Python Build & Test](https://github.com/AllenCellModeling/pylibczi/workflows/Python%20Master%20Build/badge.svg)](https://github.com/AllenCellModeling/pylibczi/actions)
-[![codecov](https://codecov.io/gh/AllenCellModeling/pylibczi/branch/feature/pybind11/graph/badge.svg)](https://codecov.io/gh/AllenCellModeling/pylibczi)
+# aicspylibczi
+[![C++ Build & Test](https://github.com/AllenCellModeling/aicspylibczi/workflows/C%2B%2B%20Master%20Build/badge.svg)](https://github.com/AllenCellModeling/aicspylibczi/actions)
+[![Python Build & Test](https://github.com/AllenCellModeling/aicspylibczi/workflows/Python%20Master%20Build/badge.svg)](https://github.com/AllenCellModeling/aicspylibczi/actions)
+[![codecov](https://codecov.io/gh/AllenCellModeling/aicspylibczi/branch/feature/pybind11/graph/badge.svg)](https://codecov.io/gh/AllenCellModeling/aicspylibczi)
```
Comment (JacksonMaxfield) [inline README.md]: This link is pointing at a previous branch instead of just at master.

Comment (JacksonMaxfield) [reply README.md]: I also don't know if the repo name gets propagated to codecov. Something to check after you merge.

File: README.md [hunk 4]
```diff
@@ -14,73 +14,104 @@ The first example show how to work with a standard CZI file (Single or Multe-Sce
...
+img, shp = czi.read_image(S=13, Z=16)
+
+# shp = [('B', 1), ('S', 1), ('C', 4), ('Z', 1), ('Y', 1300), ('X', 1900)]  # List[(Dimension, size), ...] 
+# img.shape = (1, 1, 4, 1, 1300, 1900)   # numpy.ndarray
+
+# define helper functions
+def norm_by(x, min, max):
+    norms = np.percentile(x, [min, max])
+    i2 = np.clip((x - norms[0])/(norms[1]-norms[0]), 0, 1)
+    i2
```
Comment (JacksonMaxfield) [inline README.md]: Extra i2 here that does nothing :upside_down_face:

File: README.md [hunk 5]
```diff
@@ -14,73 +14,104 @@ The first example show how to work with a standard CZI file (Single or Multe-Sce
...
 import numpy as np
-import pylibczi
+import aicspylibczi
 import pathlib
 from PIL import Image
 
-mosaic_file = pathlib.Path('~/Data/20190618_CL001_HB01_Rescan_002.czi').expanduser()
-czi = pylibczi.CziFile(mosaic_file)
+
+mosaic_file = pathlib.Path('20190618_CL001_HB01_Rescan_002.czi').expanduser()
```
Comment (JacksonMaxfield) [inline README.md]: Technically don't need the `expanduser`

File: README.md [hunk 6]
```diff
@@ -14,73 +14,104 @@ The first example show how to work with a standard CZI file (Single or Multe-Sce
...
-pip install pylibczi
+pip install aicspylibczi
 `
 
 ## Documentation
 
-[Documentation](https://pylibczi.readthedocs.io/en/latest/index.html) is available on readthedocs.
+Documentation is available at 
+[github.io](https://allencellmodeling.github.io/aicspylibczi) and 
+[readthedocs](https://aicspylibczi.readthedocs.io/en/latest/index.html).
```
Comment (JacksonMaxfield) [inline README.md]: You deploy to both?

File: aicspylibczi/CziFile.py [hunk 7]
```diff
@@ -1,18 +1,18 @@
-# This file is part of pylibczi.
+# This file is part of aicspylibczi.
 # Copyright (c) 2018 Center of Advanced European Studies and Research (caesar)
```
Comment (JacksonMaxfield) [inline aicspylibczi/CziFile.py]: I don't know if this actually applies anymore....

Comment (JacksonMaxfield) [reply aicspylibczi/CziFile.py]: We may just be able to switch it to MIT

File: aicspylibczi/CziFile.py [hunk 8]
```diff
@@ -262,21 +262,22 @@ def read_image(self, m_index: int = -1, **kwargs):
 
         Parameters
         ----------
-        m_index
+        m_index : int (optional)
```
Comment (JacksonMaxfield) [inline aicspylibczi/CziFile.py]: Could someone potentially pass both `m_index` and `M`? Would there be conflicts?

Comment (JacksonMaxfield) [reply aicspylibczi/CziFile.py]: It _may_ be worth it to simply expand all these from kwargs to full parameters. Your choice though.

File: aicspylibczi/CziFile.py [hunk 9]
```diff
@@ -285,9 +286,17 @@ def read_image(self, m_index: int = -1, **kwargs):
             sure the numpy.ndarray is interpretable. An example of the list is
                     [('S', 1), ('T', 1), ('C', 2), ('Z', 25), ('Y', 1024), ('X', 1024)]
                  so if you probed the numpy.ndarray with .shape you would get (1, 1, 2, 25, 1024, 1024).
+
         """
```
Comment (JacksonMaxfield) [inline aicspylibczi/CziFile.py]: Something about these docstring quotes seems off... It seems like there are three set's of them?

File: aicspylibczi/__init__.py [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
 #
-# pylibczi is distributed in the hope that it will be useful,
+# aicspylibczi is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with pylibczi.  If not, see <https://www.gnu.org/licenses/>.
+# along with aicspylibczi.  If not, see <https://www.gnu.org/licenses/>.
```
Comment (JacksonMaxfield) [inline aicspylibczi/__init__.py]: Again, depends on how we are moving forward with this project.

File: aicspylibczi/_version.py [hunk 11]
```diff
@@ -0,0 +1,9 @@
+# This file is part of aicspylibczi.
+#
+# You should have received a copy of the GNU General Public License
+# along with aicspylibczi.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Version information."""
+
+# The following line *must* be the last in the module, exactly as formatted:
+__version__ = "2.2.0"
```
Comment (JacksonMaxfield) [inline aicspylibczi/_version.py]: Have you already bumpversioned? If so, I would tag the commit after merging to master.

File: docs/index.rst [hunk 12]
```diff
@@ -1,5 +1,6 @@
-.. This file is part of pylibczi.
+.. This file is part of aicspylibczi.
```
Comment (JacksonMaxfield) [inline docs/index.rst]: Same license stuff.

File: setup.py [hunk 13]
```diff
@@ -1,18 +1,18 @@
...
 #
-# pylibczi is distributed in the hope that it will be useful,
+# aicspylibczi is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with pylibczi.  If not, see <https://www.gnu.org/licenses/>.
+# along with aicspylibczi.  If not, see <https://www.gnu.org/licenses/>.
```
Comment (JacksonMaxfield) [inline setup.py]: Same license stuff.

File: .github/workflows/Publish.yml [hunk 14]
```diff
@@ -1,30 +1,31 @@
...
+  wheel-setup:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.7]
+        os: [windows-latest, macOS-latest]
+
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@master
```
Comment (JacksonMaxfield) [inline .github/workflows/Publish.yml]: I'm like 99% sure you want @v1 here.

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (heeler) [reply .github/workflows/build-docs.yml]: done for whatever reason I can't get tags to work at all.

**Pull request merged**